cinema/views.py

```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```

Comment:
Code Style Issue: According to the checklist, imports should be grouped using parentheses without trailing commas for the last item. Please remove the trailing comma after 'OrderListSerializer'.